c11threads.h

```diff
@@ -122,9 +127,11 @@ static inline int mtx_init(mtx_t *mtx, int type)
 
 	pthread_mutexattr_init(&attr);
 
+#ifndef PTHREAD_MUTEX_TIMED_NP
 	if(type & mtx_timed) {
-		pthread_mutexattr_settype(&attr, PTHREAD_MUTEX_TIMED_NP);
+		pthread_mutexattr_settype(&attr, PTHREAD_MUTEX_NORMAL);
 	}
+#endif
```

Comment:
Eh ... this way we never get proper timed mutexes....
This should be:

        if(type & mtx_timed) {
    #ifdef PTHREAD_MUTEX_TIMED_NP  /* or #ifndef C11THREADS_NO_TIMED_MUTEX */
            pthread_mutexattr_settype(&attr, PTHREAD_MUTEX_TIMED_NP);
    #else
            pthread_mutexattr_settype(&attr, PTHREAD_MUTEX_NORMAL);
    #endif
        }

Reply:
ops =)